Presentation.Web/Models/API/V1/UI_Configuration/CustomizedUINodeDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Configuration;
+using System.Linq;
+using System.Web;
+
+namespace Presentation.Web.Models.API.V1.UI_Configuration
+{
```

Comment:
we are missing the top level dto .. the one with "nodes". that leaves us the flexibility to add more properties to the root level without breaking the contract with existing bindings

Reply:
Remember to align the tests and swagger doc